2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -3,21 +3,41 @@
 
 namespace ListsLogicAndIterations.Code
 {
-	public static class Dictionarys
-	{
+    public static class Dictionarys
+    {
 		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
 		{
-			throw new NotImplementedException();
+            Dictionary<Guid, decimal> result = new Dictionary<Guid, decimal>
+            {
+                { key, value }
+            };
+            return result;
 		}
 
 		public static decimal GetValue(Dictionary<Guid, decimal> dict, Guid key)
 		{
-			throw new NotImplementedException();
-		}
+            if (dict.TryGetValue(key, out _))
+            {
+                return dict[key];
+            }
+            else
+            {
+                Console.WriteLine("Value does not exist in the Dictionary");
+                return 0;
+            }
```

Comment:
Though you code works there are some things to note:

1) You are using a try get value, but you use `dict[key]` inside the if block - you are accessing your list more than onjce when you don't need to. That `out _` is giving you value when it exists so you could have just returned that. 

2) Bonus points for using a Console.WriteLine *however* if you don't have a console app (which this is not a console app) you can do Debug.WriteLine() (in System.Diagnostics) which is preferred and has more options.